src/toTransform.js

```diff
@@ -35,11 +35,58 @@
     return 'scale3d(' + valuesArray.join(', ') + ')';
   }
 
-  function toTransform (properties) {
+  function convertOffsetAnchorPosition (properties, element) {
+    /* According to spec: https://drafts.fxtf.org/motion-1/#offset-anchor-property
+       If offset-anchor is set to auto then it will compute to the value of offset-position. */
+    var position = 'auto';
+    if ('offset-position' in properties) {
+      position = internalScope.offsetPositionAnchorParse(properties['offset-position']);
+    }
+
+    if (position === 'auto' || position === undefined || position === null) {
+      return null;
+    }
+
+    var anchor = 'auto';
+    if ('offset-anchor' in properties) {
+      anchor = internalScope.offsetPositionAnchorParse(properties['offset-anchor']);
+    }
+
+    if (anchor === 'auto' || anchor === undefined || anchor === null) {
+      anchor = position;
+    }
+
+    var savedTransform = element.style._style.transform;
```

Comment:
The best thing I can come up with ATM is for the extension API to pass in the unwrapped inline style as another parameter, that way this code doesn't need to know about internal names like _style.